.env.example

```diff
@@ -1,7 +1,7 @@
-DB_NAME=dbnamehere
```

Comment:
Please re-add this environment variable.

Reply:
See comment below. I added this in the last PR in an attempt to remove your extra DB arg in db.js and this broke master as I forgot about the different database requests.